packages/e2e-tests/specs/editor/various/links.test.js

```diff
@@ -81,6 +81,30 @@ describe( 'Links', () => {
 		expect( await getEditedPostContent() ).toMatchSnapshot();
 	} );
 
+	it( 'will not pre-populate URL input if selected text is not a link', async () => {
```

Comment:
```suggestion
	it( 'will not pre-populate URL input if selected text is not a valid HTTP based URL', async () => {
```

Nit: how do you feel about making the test a little more self descriptive?